src/app/posts/components/post/post.component.scss

```diff
@@ -0,0 +1,47 @@
+.post {
+    margin: 10px;
+    &__header {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        margin: 5px;
+    }
+
+    &__author {
+        display: flex;
+        letter-spacing: 1px;
+        font-family: cursive;
```

Comment:
???

Reply:
My bad, forgot to move these text-format props to the post__author-name class, moved into the correct one in the new PR Feature/hw3.